src/hooks/useOneTxMustBeUpgraded.ts

```diff
@@ -0,0 +1,29 @@
+import { getExtensionHash } from '@colony/colony-js';
+import { InstalledExtensionData } from '~types';
+import { isInstalledExtensionData } from '~utils/extensions';
+import { useGetCurrentExtensionVersionQuery } from '~gql';
+
+export const useOneTxMustBeUpgraded = (
+  extensionData: InstalledExtensionData,
```

Comment:
Type definition needs to include the `| null` since that's how `extensionData` is defined in `ExtensionUpgrade`

Reply:
Removed useOneTxMustBeUpgraded so should no longer be relevant